### ISSUE TITLE
[Bugs] Panel buttons on top of maximized window

### ISSUE THREAD

**Issue by flickleafy** [GnomeSnapExtensions/gSnap#56]
When I occasionally need to maximize a window, in cases I need to use the entire screen, that buttons that is shown when there are many windows on a zone, is on top of the maximized window.

Probably would be a good idea to hide the buttons when a maximized window is detected.

**Comment by manfreed** [comment]
A similar, but maybe less noticeable issue happens when you move a window around, and position it to overlap the buttons. 

The tab buttons end up being over the window, even tough I would expect my focused window to be on top of everything. 

I believe the tab windows should be placed at the same Z coordinate (z-index? layer? I'm not familiar with Gnome's framework), as the topmost window of the specific tab group.

**Comment by Bennyhill500** [comment]
Yeah the Tabs being always on top over everything else really makes the whole feature useless
